lib/balena-lint.ts

```diff
@@ -308,11 +313,10 @@ export const lint = async (passedParams: any) => {
 	}
 
 	let configOverridePath;
-	// optimist converts all --no-xyz args to a argv.xyz === false
-	const prettierCheck = options.argv.prettier !== false;
-	const testsCheck = options.argv.tests === true;
-	const typescriptCheck = options.argv.typescript;
-	const autoFix = options.argv.fix === true;
+	const prettierCheck = options.argv.prettier === false ? false : true;
+	const testsCheck = options.argv.tests ? true : false;
```

Comment:
What type is `tests` here originally? I'd assume it'd already be a boolean?

Reply:
I believe so, let me double-check and fix.